lizard_auth_server/forms.py

```diff
@@ -4,7 +4,7 @@
 
 from django import forms
 from django.conf import settings
-from django.contrib import auth
+from django.contrib.auth import forms as authforms
```

Comment:
Het hangt er bij dat soort dingen van af wat er in de `__init__py` staat. 't Kan zijn dat ze daar wat hebben weggehaald/opgeschoond bij Django.

Ach, laat maar zo staan. Qua naamgeving vind ik het eigenlijk best wel netjes!